generators/firechip/src/main/scala/BridgeBinders.scala

```diff
@@ -100,13 +100,31 @@ class WithFireSimMultiCycleRegfile extends ComposeIOBinder({
   }
 })
 
+class WithGPIOTiedOff extends OverrideIOBinder({
+  (system: HasPeripheryGPIOModuleImp) =>
+    system.gpio.foreach(_.pins.foreach(_.i.ival := false.B)); Nil
+})
+
+class WithTiedOffDebug extends OverrideIOBinder({
+  (system: HasPeripheryDebugModuleImp) => {
+    Debug.tieoffDebug(system.debug, system.psd)
+    // tieoffDebug doesn't actually tie everything off :/
+    system.debug.foreach(_.clockeddmi.foreach({ cdmi => cdmi.dmi.req.bits := DontCare }))
+    Nil
+  }
+})
+
```

Comment:
Why don't the IOCell modules work with Firechip?

Reply:
I don't know exactly, but my guess is something to do with constant propagation. If there's an extra clock (e.g. the dmiClock, even if it's tied to 0) it fails during firrtl compilation (see my slack message for a stack trace).

Reply:
Ah ok. If we have to have these separate "system-level" tie-offs, instead of the "chip-level" tie-offs, I think we should reflect that in the naming, for example `WithSystemGPIOTiedOff`. We can talk about this tomorrow.

Reply:
Sure, I'm fine with that.